boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -25,7 +25,8 @@ const struct State *App_GetDriveState(void)
     static struct State drive_state = {
         .name         = "DRIVE",
         .run_on_entry = DriveStateRunOnEntry,
-        .run_on_tick  = DriveStateRunOnTick,
+        .run_on_tick_1Hz  = NULL,
+        .run_on_tick_1kHz = DriveStateRunOnTick,
```

Comment:
Maybe we can start renaming the functions in this PR? Up to you, not super for or against it.
```suggestion
        .run_on_tick_1kHz = DriveStateRunOnTick1kHz,
```

Reply:
I think I'll leave it as-is for now, as there may be things in the `RunOnTick` functions that belong in the `1Hz` task, and thus the name wouldn't be totally correct until we do an audit in a future PR to move stuff over.